iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -39,6 +46,18 @@ public String getName() {
         return name != null ? name : id;
     }
 
+    @Override
+    public boolean isFictitious() {
+        return fictitious;
+    }
+
+    @Override
+    public void setFictitious(boolean fictitious) {
+        boolean oldValue = this.fictitious;
+        this.fictitious = fictitious;
+        getNetwork().getListeners().notifyUpdate(this, "fictitious", oldValue, fictitious);
```

Comment:
You should compare old and new value and notify only if there is a modification? Check how it’s done for other attributes

Reply:
`NetworkListenerList::notifyUpdate` already check if there is a modification